packages/relay-kit/src/packs/safe-4337/Safe4337Pack.test.ts

```diff
@@ -73,30 +76,33 @@ describe('Safe4337Pack', () => {
     it('should throw an error if the 4337 Module is not enabled in the Safe account', async () => {
       await expect(
         createSafe4337Pack({
-          options: { safeAddress: fixtures.SAFE_ADDRESS_4337_MODULE_NOT_ENABLED }
+          options: { safeAddress: fixtures.SAFE_ADDRESS_4337_MODULE_NOT_ENABLED },
+          safeModulesVersion: '0.2.0'
         })
       ).rejects.toThrow(
-        'Incompatibility detected: The EIP-4337 module is not enabled in the provided Safe Account. Enable this module (address: 0xa581c4A4DB7175302464fF3C06380BC3270b4037) to add compatibility.'
+        `Incompatibility detected: The EIP-4337 module is not enabled in the provided Safe Account. Enable this module (address: ${fixtures.SAFE_4337_MODULE_ADDRESS_V0_2_0}) to add compatibility.`
       )
     })
 
     it('should throw an error if the 4337 fallbackhandler is not attached to the Safe account', async () => {
       await expect(
         createSafe4337Pack({
-          options: { safeAddress: fixtures.SAFE_ADDRESS_4337_FALLBACKHANDLER_NOT_ENABLED }
+          options: { safeAddress: fixtures.SAFE_ADDRESS_4337_FALLBACKHANDLER_NOT_ENABLED },
+          safeModulesVersion: '0.2.0'
         })
       ).rejects.toThrow(
-        'Incompatibility detected: The EIP-4337 fallbackhandler is not attached to the Safe Account. Attach this fallbackhandler (address: 0xa581c4A4DB7175302464fF3C06380BC3270b4037) to ensure compatibility.'
+        `Incompatibility detected: The EIP-4337 fallbackhandler is not attached to the Safe Account. Attach this fallbackhandler (address: ${fixtures.SAFE_4337_MODULE_ADDRESS_V0_2_0}) to ensure compatibility.`
       )
     })
 
-    it('should throw an error if the Safe Modules do not match the supported version', async () => {
+    it('should throw an error if the 4337 Module is not enabled in the Safe account', async () => {
```

Comment:
This title was changed but the test still checks the same
```suggestion
    it('should throw an error if the Safe Modules do not match the supported version', async () => {
```